content/patterns/cockroach/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Cockroach
```

Comment:
Should this directory be `cockroachdb` to align with https://github.com/hybrid-cloud-patterns/cockroachdb-pattern? (Commenting of this line since I can't add one inline for the directory name)